### PR TITLE
Fixed #87 parse URL regex now works with Chrome apps scheme.

### LINKED ISSUE #87: Regex parsing the URL is not handing Chrome Apps
Regex defined in history.js in line [416](https://github.com/devote/HTML5-History-API/blob/master/history.js#L416) do not parse `chrome-extension://` protocol properly.

Chrome apps don't have history API and for some libraries to work the app need to include a polyfil.

## Expected outcome
Function `parseURL()` returns: 
`_href` - without `undefined` values
`protocol` - is `chrome-extension://`
`host` - is not undefined
`_hostname` - is not undefined

## Actual outcome
The `result` variable - after parsing the URL - have following values:
```javascript
[
  "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  null,
  null,
  null,
  "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  null,
  null
]
```
So in case of my application result of calling this function is:
```javascript
{
  "_href": "undefined//undefinedchrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  "_host": "undefined",
  "_port": "",
  "_pathname": "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  "_search": "",
  "_hash": "",
  "_relative": "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  "_nohash": "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport",
  "_special": "chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/!dataimport"
}
```
## Suggested solution
Replace first `\w+` in the `re` regex in referenced line to characters range for all letters and dash: `[a-zA-Z\-]+`. So the final regex would be: 
`/(?:([a-zA-Z\-]+\:))?(?:\/\/(?:[^@]*@)?([^\/:\?#]+)(?::([0-9]+))?)?([^\?#]*)(?:(\?[^#]+)|\?)?(?:(#.*))?/`

### PR DESCRIPTION
**Pull request by jarrodek**

### CODE REVIEW

**Comment by devote** [comment]
Hi,
Scheme may also contain digits. What is the cause of your changes?

**Comment by jarrodek** [comment]
I've updated regex so it will allow digits in schema.

File: package.json [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
+		"pushstate",
+		"replacestate",
+		"hashes",
+		"hashbang",
+		"history",
+		"html5",
+		"devote"
+	],
+	"licenses": [
+		{},
```
Comment (devote) [inline package.json]: Why have you removed the license information?

Comment (jarrodek) [inline package.json]: Hmm, I haven't noticed it. I'm using some new add-on for Atom and it rewrites packages.json files as soon as I open it. I guess it deleted the line. I'll restore it in a moment.

**Comment by devote** [comment]
Good job, Thanks!

**Pull request merged**